app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -9,6 +9,10 @@ module TwoFactorAuthenticatable
   ALLOWED_OTP_DRIFT_SECONDS = 30
   DIRECT_OTP_VALID_FOR_MINUTES = IdentityConfig.store.otp_valid_for
   DIRECT_OTP_VALID_FOR_SECONDS = DIRECT_OTP_VALID_FOR_MINUTES * 60
+  OTP_FORMAT = {
+    digit: I18n.t('telephony.format_type.digit'),
+    char: I18n.t('telephony.format_type.char'),
+  }
```

Comment:
Are there significant benefits to using the constant here vs. putting the translation directly in the call?

Reply:
Also, I don't think we can have translated values as constants of a module, since I don't think they would incorporate the current user's locale.

Reply:
ah good point